tests/cps_agg_expected.txt

```diff
@@ -25,8 +25,8 @@ n21               711316
 ssi_ben           103019670
 vet_ben           469788696
 snap_ben          141713659
-mcare_ben         1700697749
-mcaid_ben         904042846
+mcare_ben         1778073024
+mcaid_ben         888211102
```

Comment:
Are these two the total amounts of benefits for medicare and medicaid respectively?

Reply:
These are the unweighted totals, yes. The weighted totals didn't change

Reply:
Got you.